### PR TITLE
[@testing-library/dom] v6.11 - add new history popstate event

### PR DESCRIPTION
**Pull request by bochap**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/testing-library/dom-testing-library/pull/433>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/testing-library__dom/events.d.ts [hunk 1]
```diff
@@ -80,7 +80,8 @@ export type EventType =
     | 'pointerOut'
     | 'pointerLeave'
     | 'gotPointerCapture'
-    | 'lostPointerCapture';
+    | 'lostPointerCapture'
+    | 'popstate';
```
Comment (kentcdodds) [inline types/testing-library__dom/events.d.ts]: Hmmm... Shouldn't this be `popState`?

Comment (weyert) [reply types/testing-library__dom/events.d.ts]: I had a look earlier but https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onpopstate seems to suggest it is `popstate`?

Comment (weyert) [reply types/testing-library__dom/events.d.ts]: Also they seem to  use `popstate` in JSDOM [1] so I think it's correct?

[1] https://github.com/jsdom/jsdom/blob/d6f8a97b5fb7709d6ad0215c1ae95fd4cab58489/lib/jsdom/living/nodes/WindowEventHandlers-impl.js#L18

**Review by weyert — APPROVED** [review]

File: types/testing-library__dom/events.d.ts (hunk 1 above)
Comment (weyert) [reply types/testing-library__dom/events.d.ts]: Wait, there is also this PR for `dom-testing-library` which uses `popState`: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41951/files 

I am confused now

Comment (eps1lon) [reply types/testing-library__dom/events.d.ts]: [It is `popState`](https://github.com/testing-library/dom-testing-library/pull/433/files#diff-4607e46b15da8ae44c4b47730c6d5963R342)

**Review by eps1lon — CHANGES_REQUESTED** [review]
Seems like we were working on the same things. I very much appreciate the work but this was already implemented in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41951

File: types/testing-library__dom/events.d.ts (hunk 1 above)
Comment (seetdev) [reply types/testing-library__dom/events.d.ts]: Thanks I will cancel this PR since there is another one. And I got it wrong it should be popState

**Pull request closed without merging**